sig-architecture/api-review-process.md

```diff
@@ -27,39 +27,37 @@ Ideally, those whose API review priority is shifted in a release-impacting way w
 
 # What APIs need to be reviewed?
 
-* What are the kind of reviews?
+## Mandatory
+The changer is expected to submit the changes to API review, and the change is blocked on the API reviewer approval.  As a general rule of thumb, anything that is considered to be part of "core Kubernetes" requires a mandatory API review.
 
-    * **mandatory**: The changer is expected to submit the changes to API review, and the change is blocked on the API reviewer approval 
+* All API implementations (including alpha versions) that are part of core Kubernetes must be reviewed, including CRDs, so user experience across the Kubernetes ecosystem is consistent.
 
-    * **voluntary**: Changer can request a review, or a 3rd party can nominate the API for review.  The review comments are recommendations.
+* github.com/kubernetes/kubernetes, and any kubernetes-* org projects it depends on are required to be reviewed.
 
-* Which projects are in scope?
+* Kubernetes-style API that uses a *.k8s.io or *.kubernetes.io name, e.g. "storage.k8s.io" that are not listed in the [voluntary section](#voluntary) below.
 
-    * github.com/kubernetes/kubernetes, and any kubernetes-* org projects it depends on are required to be reviewed.   (mandatory)
+* "Critical" or other “highly-integrated” APIs, such as our extension points in the node, apiserver, and controllers.  This includes CSI, CNI, CRI, and CPI.
 
-    * Any kubernetes-style API that uses a *.k8s.io or *.kubernetes.io name, e.g. "storage.k8s.io" (mandatory)
+## Voluntary
+Voluntary reviews apply towards non-core APIs that do not meet the [mandatory](#mandatory) requirements listed above.  Changer can request a review, or a 3rd party can nominate the API for review.  The review comments are considered recommendations.    
 
-    * "Critical" or other “highly-integrated” APIs, such as our extension points in the node, apiserver, and controllers as defined by SIG Architecture [TBD].  (mandatory)
+* SIG sponsored CRD based APIs outside of the core that use the "*.extension.k8s.io" namespace.
```

Comment:
Happy to entertain other namespaces here, but I thought this one sounded good, and made it pretty clear.

Reply:
As much as I agree with wanting to make it clear what is and is-not part of the system, this feels a bit like it is creating a ghetto.  

If we want to be clear in terms of names I see 4 paths:

* API review all of *.k8s.io with particular guidelines for non-core APIs, and no visibility to end-users.
* Carve out a sub-space for extensions and experiments (e.g. x.k8s.io or ext.k8s.io)
* Get one new domain for these and put all non-core APIs there
* Get a new domain for each effort

Reply:
My objective is to be explicit, like we were when we created the org structure.  Right now it's worse than a ghetto, it's the wild west.